contrib/depends/packages/packages.mk

```diff
@@ -1,6 +1,14 @@
+ifeq ($(host_os),android)
+packages:=boost openssl zeromq cppzmq libiconv
+else
```

Comment:
Was thinking of moving expat, ldns, and the hardware wallet support libs to another macro instead, and keep
packages as just the bare minimum set.

Reply:
... can revisit this at a later date to see if the hardware wallet stuff will build for Android. Just ran out of time for this go round.